src/Servers/IIS/IIS/test/IIS.ShadowCopy.Tests/IIS.ShadowCopy.Tests.csproj

```diff
@@ -0,0 +1,48 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <TestGroupName>IIS.ShadowCopy.Tests</TestGroupName>
+    <DisableFastUpToDateCheck>True</DisableFastUpToDateCheck>
+    <SkipTests Condition=" '$(SkipIISTests)' == 'true' ">true</SkipTests>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+
+  <Import Project="../FunctionalTest.props" />
+
+  <ItemGroup>
+    <ProjectReference Include="..\testassets\IIS.Common.TestLib\IIS.Common.TestLib.csproj" />
+    <ProjectReference Include="..\testassets\InProcessWebSite\InProcessWebSite.csproj"
+      Private="false"
+      ReferenceOutputAssembly="false"
+      SkipGetTargetFrameworkProperties="true" />
+  </ItemGroup>
+
+  <ItemGroup>
```

Comment:
These item groups are getting cumbersome. Suggest making a single "infrastructure++" project and have it share `internal`s with the real test projects. Ideal if that let's us move some of the code out of `$(SharedSourceRoot)`.

Reply:
I'll file a tracking item for that suggestion, but I'm not planning on doing that refactoring as part of this PR

Reply:
Filed https://github.com/dotnet/aspnetcore/issues/38608

Reply:
nit: The new issue is great. Could some of these move into ../FunctionalTest.props in the meantime❔

Reply:
Not in this PR, its not a trivial refactoring, I'll include this in things tracked in the issue.